python-sdk/tests_integration/astro_deploy/master_dag.py

```diff
@@ -52,30 +52,59 @@ def get_report(dag_run_ids: List[str], **context: Any) -> None:  # noqa: C901
                 f"Airflow version for the below astro-sdk run is `{airflow_version}` "
                 f"with {os.environ['AIRFLOW__CORE__EXECUTOR']} \n\n"
             )
-        message_list.append(airflow_version_message)
+        master_dag_deployment_link = (
+            f"{os.environ['AIRFLOW__WEBSERVER__BASE_URL']}"
+            f"/dags/example_master_dag/grid?search=example_master_dag"
+        )
+
+        deployment_message = (
+            f"\n <{master_dag_deployment_link}|Link> "
+            f"to the master DAG for the above run on Astro Cloud deployment \n"
+        )
 
+        dag_count, failed_dag_count = 0, 0
         for dr in last_dags_runs:
             dr_status = f" *{dr.dag_id} : {dr.get_state()}* \n"
-            message_list.append(dr_status)
+            dag_count += 1
+            failed_tasks = []
             for ti in dr.get_task_instances():
                 task_code = ":black_circle: "
                 if ti.task_id not in ["end", "get_report"]:
                     if ti.state == "success":
-                        task_code = ":large_green_circle: "
+                        if ti.state == "success":
```

Comment:
Why do we have two lines checking for the same thing?

Reply:
@tatiana rightly pointed, will fix this